lib/std/math/big/int_test.zig

```diff
@@ -2799,7 +2800,7 @@ test "big int conversion read twos complement with padding" {
     // (2) should correctly order bytes based on the provided endianness
     // (3) should sign-extend any bits from bit_count to 8 * abi_size
 
-    var bit_count: usize = 12 * 8 + 1;
+    const bit_count: usize = 12 * 8 + 1;
```

Comment:
Also this doesn't seem right. A few lines down in line 2815 this variable seems to be modified:
```zig
bit_count = 12 * 8 + 2;
```